packages/medusa/src/api-v2/admin/promotions/rule-operator-options/route.ts

```diff
@@ -0,0 +1,32 @@
+import { PromotionRuleOperator } from "@medusajs/utils"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../../types/routing"
+
+export const GET = async (
+  req: AuthenticatedMedusaRequest,
+  res: MedusaResponse
+) => {
+  const operators = [
```

Comment:
suggestion: since this is a const, maybe we can extract it from the function

Reply:
@adrien2p addressed all your comments. 💪🏻